packages/core/src/domain/session/sessionStore.ts

```diff
@@ -79,7 +79,28 @@ export function startSessionStore<TrackingType extends string>(
   }
 
   function isSessionInCacheOutdated(cookieSession: SessionState) {
-    return sessionCache.id !== cookieSession.id || sessionCache[productKey] !== cookieSession[productKey]
+    if (sessionCache.id !== cookieSession.id) {
+      if (cookieSession.id) {
```

Comment:
Could you explain this condition? I'm not sure to see why we shouldn't display a message when `cookieSession.id` is undefined.

Reply:
so I see 3 cases:

1. cached id x / stored id y => unexpected change of id
2. cached id undefined / stored id defined => can be new session created in another tab
3. cached id defined / stored id undefined => can be the expiration of the cookie

I should indeed add an extra condition to avoid to report messages for case 2

Reply:
Oh ok. I think it's fine for case 2, since `undefined === undefined`, it won't enter the first `if`. Maybe add a small comment ? `// if cookie session id is undefined, could be the expiration of the cookie`

Reply:
about case 2, it would enter the first if since one id is defined and not the other.
However, it does mean that a session is created in another tab, this behavior would be from state: no session in cache / stored id undefined.
So it seems all good.
I've added a comment